src/ddmd/dclass.d

```diff
@@ -855,26 +865,59 @@ extern (C++) class ClassDeclaration : AggregateDeclaration
             auto s = (*members)[i];
             if (s.apply(&func, cast(void*)this))
             {
-                isabstract = ABSyes;
-                return true;
+                return yes();
+            }
+        }
+
+        /* If the base class is not abstract, then this class cannot
+         * be abstract.
+         */
+        if (!baseClass || !baseClass.isAbstract())
+            return no();
```

Comment:
This causes another regression.  This routine now returns `false` if called with an interface.

e.g:
```
interface A { void foo(); }
interface AB : A {}
```
GDC now fails to compile this simply because it calls `fillVtbl` during the codegen stages, which results in the error `interface function is not implemented`.